CDCgov/prime-reportstream
# Jessica/10370 orm mapping

This PR ...

Adds/updates the necessary resource files to get the information from an ORM_O01 HL7 V2 message to a FHIR message. 

Test Steps:
1. Reach out to me (Jessica) to get the sample ORM.hl7 message
2. Run something like ./prime fhirdata -i prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/orm.hl7 -o prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/orm.fhir  --output-format FHIR to convert it to FHIR
3. Check that the HL7 fields all make it over to the FHIR message.

## Changes
Updated resources to have the changes needed to transfer all ORM data to FHIR

## Checklist

### Testing
